### PR TITLE
[cleanup] remove unused functions

### PR DESCRIPTION
**Pull request by hezhizhen**
## Summary

Title says it.

### CODE REVIEW

File: internal/services/services.go [hunk 1]
```diff
@@ -35,85 +28,6 @@ const (
 	stopService
 )
 
-func listenToAutoPortForwardingChangesOnRemote(
```
Comment (savil) [inline internal/services/services.go:38]: @mikeland86 any plans for this?

Comment (mikeland86) [reply internal/services/services.go:38]: let's kill it!

This was meant to auto-forward ports between devbox cloud and your local machine but in practice it's brittle compared to our existing service proxy.

File: internal/services/services.go [hunk 2]
```diff
@@ -35,85 +28,6 @@ const (
...
-					color.New(color.FgYellow).Fprintf(w, "Port forwarding %s:%s -> localhost stopped\n", hostname, service.Port)
-					cancel()
-				}
-				return service, false
-			},
-		},
-	)
-}
-
-func printProxyURL(w io.Writer, services Services) error { // TODO: remove it?
```
Comment (mikeland86) [inline internal/services/services.go:82]: Looks like the callsite for this got accidentally removed in https://github.com/jetpack-io/devbox/pull/836

cc: @Lagoja 

This function used to be called to print the proxy URL that users can use to access their services. Without it users won't know how to access their services on the cloud.

File: internal/services/services.go (hunk 1 above)
Comment (hezhizhen) [reply internal/services/services.go:38]: Okay.

**Review by savil — APPROVED** [review]

**Pull request merged**